### PR TITLE
move process readme to main folder

### PR DESCRIPTION
**Pull request by teemow**
the rfc-process folder won't be obvious in the future. so i thought it is maybe better to have the process description in the main readme

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,35 @@
-# rfc
-Our requests for comments
+# RFC Process
+
+This RFC describes the RFC process for Giant Swarm.
+
+Giant Swarm is a growing, distributed company, and we want to ensure that everyone continues to have the ability to bring their ideas and thoughts into the hivemind. While a large amount of our discussion currently happens successfully in chats and calls, a long-form writing process may provide some benefits: i.e: promoting deep thinking and discussion, creating a record of considered ideas and discussion, reducing number of meetings (and allowing easier management of timezones), and providing avenues for increasing the asynchronousity of our work.
```
Comment (stone-z) [inline README.md:5]:
```suggestion
Giant Swarm is a growing, distributed company, and we want to ensure that everyone continues to have the ability to bring their ideas and thoughts into the hivemind. While a large amount of our discussion currently happens successfully in chats and calls, a long-form writing process may provide some benefits: i.e: promoting deep thinking and discussion, creating a record of considered ideas and discussion, reducing number of meetings (and allowing easier management of timezones), and providing avenues for increasing the asynchronicity of our work.
```

File: README.md [hunk 2]
```diff
@@ -1,2 +1,35 @@
-# rfc
-Our requests for comments
+# RFC Process
```
Comment (JosephSalisbury) [inline README.md:1]:
```suggestion
# rfc
```

File: README.md [hunk 3]
```diff
@@ -1,2 +1,35 @@
...
+
+This RFC describes the RFC process for Giant Swarm.
```
Comment (JosephSalisbury) [inline README.md:3]:
```suggestion
This describes the RFC process for Giant Swarm.
```

**Review by stone-z — APPROVED** [review]
One nit, otherwise LGTM

**Review by JosephSalisbury — APPROVED** [review]
i like it - few suggestions for making it more of a README for the repo

**Pull request merged**